pkg/scaling/executor/scale_jobs_test.go

```diff
@@ -299,11 +357,15 @@ type pendingJobTestData struct {
 }
 
 func getMockScaleExecutor(client *mock_client.MockClient) *scaleExecutor {
+	scheme := runtime.NewScheme()
+	kedav1alpha1.AddToScheme(scheme)
+	clientgoscheme.AddToScheme(scheme)
```

Comment:
```suggestion
	utilruntime.Must(kedav1alpha1.AddToScheme(scheme))
	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
```
with importing `utilruntime "k8s.io/apimachinery/pkg/util/runtime"` is a common way to check the return value